src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -18,7 +19,7 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <DebugType>embedded</DebugType>
     <!-- See https://github.com/Azure/azure-functions-durable-extension/issues/1433 -->
-    <NoWarn>NU5125;SA0001</NoWarn>
+    <NoWarn>NU5125;SA0001;NU1507</NoWarn>
```

Comment:
This change should be removed because currently we have to use two feeds to test the private DTFx.AzureStorage v2.4.0-encdoing.1 package.  When v2.4.0 of DTFx.AzureStorage  is officially released and updated in Directory.Package.props, this needs to be removed.